### ISSUE TITLE
Failure to install certbot on Xenial from PPA

### ISSUE THREAD

**Issue by kalxas** [certbot/certbot#5247]
## My operating system is (include version):
Ubuntu Server 16.04 Xenial

## I installed Certbot with (certbot-auto, OS package manager, pip, etc):
Certbot PPA:
https://launchpad.net/~certbot/+archive/ubuntu/certbot

## I ran this command and it produced this output:
```
$ sudo apt-get install python-certbot-nginx
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 python-certbot-nginx : Depends: certbot (>= 0.19.0~) but it is not going to be installed
                        Depends: python-certbot but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```

## Certbot's behavior differed from what I expected because:
On my staging server, the above command worked some weeks ago.

**Comment by kalxas** [comment]
```
$ sudo apt-get install python-dialog
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 python-dialog : Depends: dialog but it is not installable
E: Unable to correct problems, you have held broken packages.
```

**Comment by kalxas** [comment]
Hi @oerdnj
I see that the package python2-pythondialog was backported but the dialog dependency was not removed (since it is not available on Xenial)
Changelog:
```
python2-pythondialog (3.3.0-2+certbot~xenial+1) xenial; urgency=medium

  * Backport for Xenial Xerus (16.04 LTS)

 -- Ondřej Surý <ondrej@debian.org>  Wed, 02 Nov 2016 16:24:42 +0100

python2-pythondialog (3.3.0-2) unstable; urgency=medium

  * Add dependency on dialog (closes: #806371)
  * Move maintenance of this package to the letsencrypt team

 -- Francois Marier <francois@debian.org>  Thu, 26 Nov 2015 14:23:32 -0800
```
Control:
```
Package: python-dialog
Architecture: any
Depends: dialog, ${misc:Depends}, ${python:Depends}
```

**Comment by kalxas** [comment]
The multiuniverse repository was not enabled on that server, sorry for the noise.
